### DISCUSSION TITLE
Manually Stack multiple Images

### DISCUSSION THREAD

**Discussion by mirisbowring** [photoprism/photoprism#2485] [Q&A] [answered]
Hi,

i did not find an answer in the docs.
I have about 30 Fotos that have been shot within a minimal timeframe to capture a quick movement.
To tidy things up, i would like to stack them manually.

How can I achieve my desire?

Regards

**Comment by graciousgrey** [comment, marked as answer]
Manual stacking is not implemented yet: https://github.com/photoprism/photoprism/issues/28

This idea might also be interesting to you: https://github.com/photoprism/photoprism/issues/1446
